tests/rados/test_node_drain_customer_bug.py

```diff
@@ -194,7 +200,7 @@ def run(ceph_cluster, **kw):
             return 1
 
         if bug_exists:
-            active_osd_list = rados_obj.get_osd_list(status="up")
+            active_osd_list = rados_obj.get_active_osd_list()
```

Comment:
Please revert to using get_osd_list(status="up")

Reply:
Done.